### DISCUSSION TITLE
Share stats with someone

### DISCUSSION THREAD

**Discussion by bzoum** [plausible/analytics#775] [Anything Else]
Hello,
I know that we can share all the stats of the website by making them public.
I would like to make only one part of the stats public.
Use case: I wrote an article to promote an artist. I want to share with him the stats related to this page. And not the others.
Is it possible?

**Comment by metmarkosaric** [comment]
thanks @bzoum! sharing stats for one individual page on a website is not possible unfortunately. you can create [a private and secure link](https://plausible.io/docs/shared-links) to share the website stats but that is for the whole site. otherwise perhaps a screenshot would work for this use case at the moment?

**Reply by bzoum** [reply]
Thank you. I don't want to share the whole stats.
And the screenshot is not a good option because it's just a screenshot...
Maybe, it could be a feature request? (if it's technically feasible??).

**Reply by metmarkosaric** [reply]
it's something we can consider to do in the future. alternative that's possible to do at the moment but not as easy and convenient is for you to create a little dashboard for that specific page using [our api](https://plausible.io/docs/stats-api).
